Session/Meta/BUILDING.md

```diff
@@ -0,0 +1,91 @@
+# Building
```

Comment:
Looks like I added a duplicate `BUILDING.md` file in the YYImage PR - will remove this there